src/tooltip/model.ts

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
This whole file is gone now in `master`. What's the best way to keep the new pattern in master? Since there's no rebase required for this PR to go in, I'm not sure.